ui/decredmaterial/editor.go

```diff
@@ -26,6 +30,31 @@ type Editor struct {
 	shaper text.Shaper
 }
 
+type EditorCustom struct {
+	hint       string
+	TitleLabel Label
+	ErrorLabel Label
+	LineColor  color.RGBA
+
+	editorMaterial Editor
+	flexWidth      float32
+	editor         *widget.Editor
+	//IsVisible if true, displays the paste and clear button.
+	IsVisible bool
+	//IsRequired if true, displays a required field text at the buttom of the editor.
+	IsRequired bool
+	// SingleLine force the text to stay on a single line.
+	// SingleLine also sets the scrolling direction to
+	// horizontal.
+	SingleLine bool
+
+	pasteBtnMaterial IconButton
+	pasteBtnWidget   *widget.Button
+
+	clearBtMaterial IconButton
+	clearBtnWidget  *widget.Button
+}
+
```

Comment:
Is there a reason we have an Editor and a Custom Editor? Why not have just one editor?